src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,142 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size = 0;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
+        T item;
+        Node<T> next;
+        Node<T> prev;
+
+        Node(Node<T> prev, T element, Node<T> next) {
+            this.item = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        final Node<T> l = last;
+        final Node<T> newNode = new Node<>(l, value, null);
+        last = newNode;
+        if (l == null) {
+            first = newNode;
+        } else {
+            l.next = newNode;
+        }
+        size++;
+        return true;
     }
 
     @Override
     public void add(T value, int index) {
-
+        checkIndex(index, size);
+        if (index == size) {
+            add(value);
```

Comment:
```suggestion
            add(value);
            return;
```
і тепер можна видалити один блок else